eshop/apps/feedbacks/tests/tests_feedback.py

```diff
@@ -49,12 +42,95 @@ def test_list_no_empty(self):
             "results": [
                 {
                     "id": 1,
-                    "user": "admin",
-                    "user_email": "admin@mail.com",
+                    "user": "odin",
+                    "user_email": "odin@email.com",
                     "user_phone": '+380320987654',
                     "subject": "DP",
                     "feedback": "How are you?",
                     "is_published": True
                 }
             ]
         })
+
+    def tests_create_feedback_from_superuser(self):
+        self.c.force_login(self.super_admin)
+        response = self.c.post('/feedback/', data={
+            "user_phone": "+380981234567",
+            "subject": 'DP',
+            "feedback": "test feedback from superuser",
+            "is_published": True
+        }, format='json')
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+    def tests_create_feedback_from_user(self):
+        self.c.force_login(self.no_admin)
+        response = self.c.post('/feedback/', data={
+            "user_phone": "+380981234567",
+            "subject": 'DP',
+            "feedback": "test feedback from superuser",
+            "is_published": True
+        }, format='json')
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+    def tests_delete_not_own_feedback_from_user(self):
+        self.c.force_login(self.super_admin)
+        self.c.post('/feedback/', data={
```

Comment:
feedback = Feedback.objects.create(user=self.super_admin, user_phone='+380320987654',
                                subject='DP',
                                feedback='How are you?',
                                is_published=True,
                                user=self.super_admin)